webapp/packages/core-blocks/src/ItemList/ItemListSearch.tsx

```diff
@@ -67,9 +74,9 @@ export const ItemListSearch: React.FC<IProps> = function ItemListSearch({ value,
           onChange={changeHandler}
           {...use({ mod: 'surface' })}
         />
-        <search-button as="div" onClick={searchHandler}>
-          <ListSearchButton name="search" />
-        </search-button>
+        <action-button as="div">
+          {!onSearch && (value ?? search) ? <IconButton name="cross" onClick={resetSearch} /> : <IconButton name="search" onClick={searchHandler} />}
+        </action-button>
```

Comment:
why you use `onSearch` in condition? You need to:
```js
const inputValue = value ?? search;
...
{inputValue ? ... : ... }
```

Reply:
Cause if we have onSearch handler, that means we can enter a string f.e. 'localhost' and click the search icon. We don't display cross button in this case.